### ISSUE TITLE
Option for a more balanced starting situation

### ISSUE THREAD

**Issue by d-albrecht** [Sesu8642/FeudalTactics#25]
I have read that you like to challenge yourself with almost impossible looking games, but I would prefer if the game would generate more fair start points. I know that I have the option to regenerate the map, but to me it would appeal more if a higher percentage of the generated games would immediately match my taste of difficulty. For example, it isn't uncommon that I only get one kingdom with only two tiles and all other tiles of my color are spread out across the map (on small maps). Usually with another player nearby that has a kingdom of four tiles or more. This other player could in theory eliminate me before I even get a chance of playing.

And I know that fairness is hard to define. But one metric could be number of territories in some kingdoms. So, in the example above, only two of my tiles would count as "in kingdom" while the others are effectively dead. Another metric could be "size of largest kingdom". All of these metrics could have some margin (either absolute or percentage based) as it will not be possible to generate only fair game if the fairness criteria are too narrow.

If we got a general settings menu, this could either be a simple on/off-switch or a slider denoting the acceptable margin. I read that you don't want to add too many new options to the game selector screen and that's a reasonable stance, so - also given my other ideas - a general game settings menu would really help with this and other ideas to let the player set parameters that would apply to any generated game.

**Comment by Sesu8642** [comment]
Right now, there is one mechanic that tries to balance it at least a little. The players are ordered by the number of "in kingdom" tiles they have. So if you only get a single kingdom with two tiles, there is a high chance that you will go first and can attack the larger kingdoms before they have a chance to do anything.

That said, some configuration option regarding fairness sounds like a good idea.

**Comment by d-albrecht** [comment]
Yes, but even if I can cut the other kingdom in half, the bot will still start with significantly more money if I'm not mistaken. So, sure, this is a good attempt to even out the fairness a bit, but if the distribution is totally rigged, you can only affect so much with the player order.
